hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OzoneContainer.java

```diff
@@ -223,6 +225,8 @@ public OzoneContainer(
     Duration blockDeletingSvcInterval = conf.getObject(
         DatanodeConfiguration.class).getBlockDeletionInterval();
 
+    checksumTreeManager = new ContainerChecksumTreeManager(config);
```

Comment:
We also need to stop/shutdown this manager to unregister the metrics added in #6864

Reply:
Added a workaround for now. This will be properly handled in [HDDS-10239](https://issues.apache.org/jira/browse/HDDS-10239)